app/src/ui/upstream-already-exists/upstream-already-exists.tsx

```diff
@@ -52,14 +52,17 @@ export class UpstreamAlreadyExists extends React.Component<
       >
         <DialogContent>
           <p>
-            <Ref>{name}</Ref> is a fork of <Ref>{parentName}</Ref>, but its{' '}
-            <Ref>{UpstreamRemoteName}</Ref> remote is <Ref>{existingURL}</Ref>{' '}
-            instead of <Ref>{replacementURL}</Ref>
+            The repository <Ref>{name}</Ref> is a fork of{' '}
+            <Ref>{parentName}</Ref>, but its <Ref>{UpstreamRemoteName}</Ref>{' '}
+            remote points elsewhere.
           </p>
           <p>
-            Would you like to update the <Ref>{UpstreamRemoteName}</Ref> remote
-            to use the expected URL?
+            • Current: <Ref>{existingURL}</Ref>
```

Comment:
Shouldn’t this use a `<ul>`?

```jsx
<ul>
  <li>Current: <Ref>{existingURL}</Ref></li>
  <li>Expected: <Ref>{replacementURL}</Ref></li>
</ul>
```

Reply:
Yup, good catch 👍 I forgot to update it after yolo'ing the message.